src/client/Cargo.toml

```diff
@@ -48,7 +48,12 @@ dataplane = { version = "0.5.1", path = "../dataplane-protocol", package = "fluv
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 dirs = "1.0.2"
+
+[target.'cfg(unix)'.dependencies]
 fluvio-spu-schema = { version = "0.6.1", path = "../spu-schema", features = ["file"] }
+[target.'cfg(windows)'.dependencies]
+fluvio-spu-schema = { version = "0.6.1", path = "../spu-schema" }
```

Comment:
we do need file support otherwise can't load it config

Reply:
You mean the config in `~/.fluvio/config` (or whatever in windows)? I think that part works here.

Reply:
ok